CRITICAL - 2021-02-01 10:04:10 --> Nenhuma conexão pôde ser feita porque a máquina de destino as recusou ativamente.

#0 C:\xampp\htdocs\nxgestao\system\Database\BaseConnection.php(377): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\nxgestao\system\Database\BaseConnection.php(643): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\nxgestao\system\Database\BaseBuilder.php(1845): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\xampp\htdocs\nxgestao\system\Model.php(489): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xampp\htdocs\nxgestao\app\Controllers\Login.php(45): CodeIgniter\Model->first()
#5 C:\xampp\htdocs\nxgestao\system\CodeIgniter.php(914): App\Controllers\Login->index()
#6 C:\xampp\htdocs\nxgestao\system\CodeIgniter.php(400): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
#7 C:\xampp\htdocs\nxgestao\system\CodeIgniter.php(308): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\nxgestao\public\index.php(45): CodeIgniter\CodeIgniter->run()
#9 {main}
CRITICAL - 2021-02-01 10:35:18 --> Undefined index: CNPJ
#0 C:\xampp\htdocs\nxgestao\app\Controllers\NFe.php(1066): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Undefined index...', 'C:\\xampp\\htdocs...', 1066, Array)
#1 C:\xampp\htdocs\nxgestao\system\CodeIgniter.php(914): App\Controllers\NFe->NFeAvulsaSaida()
#2 C:\xampp\htdocs\nxgestao\system\CodeIgniter.php(400): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\NFe))
#3 C:\xampp\htdocs\nxgestao\system\CodeIgniter.php(308): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\xampp\htdocs\nxgestao\public\index.php(45): CodeIgniter\CodeIgniter->run()
#5 {main}
CRITICAL - 2021-02-01 10:35:28 --> Undefined index: CEP
#0 C:\xampp\htdocs\nxgestao\app\Controllers\NFe.php(1071): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Undefined index...', 'C:\\xampp\\htdocs...', 1071, Array)
#1 C:\xampp\htdocs\nxgestao\system\CodeIgniter.php(914): App\Controllers\NFe->NFeAvulsaSaida()
#2 C:\xampp\htdocs\nxgestao\system\CodeIgniter.php(400): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\NFe))
#3 C:\xampp\htdocs\nxgestao\system\CodeIgniter.php(308): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\xampp\htdocs\nxgestao\public\index.php(45): CodeIgniter\CodeIgniter->run()
#5 {main}
CRITICAL - 2021-02-01 10:36:38 --> Undefined index: CPF
#0 C:\xampp\htdocs\nxgestao\app\Controllers\NFe.php(1068): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Undefined index...', 'C:\\xampp\\htdocs...', 1068, Array)
#1 C:\xampp\htdocs\nxgestao\system\CodeIgniter.php(914): App\Controllers\NFe->NFeAvulsaSaida()
#2 C:\xampp\htdocs\nxgestao\system\CodeIgniter.php(400): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\NFe))
#3 C:\xampp\htdocs\nxgestao\system\CodeIgniter.php(308): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\xampp\htdocs\nxgestao\public\index.php(45): CodeIgniter\CodeIgniter->run()
#5 {main}
CRITICAL - 2021-02-01 10:47:38 --> Undefined variable: dados_do_destinatario
#0 C:\xampp\htdocs\nxgestao\app\Controllers\NFe.php(779): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Undefined varia...', 'C:\\xampp\\htdocs...', 779, Array)
#1 C:\xampp\htdocs\nxgestao\app\Controllers\NFe.php(1470): App\Controllers\NFe->montaXmlSaidaNFeAvulsa(Array, Array, Array)
#2 C:\xampp\htdocs\nxgestao\system\CodeIgniter.php(914): App\Controllers\NFe->NFeAvulsaSaida()
#3 C:\xampp\htdocs\nxgestao\system\CodeIgniter.php(400): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\NFe))
#4 C:\xampp\htdocs\nxgestao\system\CodeIgniter.php(308): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#5 C:\xampp\htdocs\nxgestao\public\index.php(45): CodeIgniter\CodeIgniter->run()
#6 {main}
CRITICAL - 2021-02-01 10:48:07 --> syntax error, unexpected '$ide' (T_VARIABLE)
#0 C:\xampp\htdocs\nxgestao\system\Router\Router.php(192): CodeIgniter\Router\Router->autoRoute('NFe/NFeAvulsaSa...')
#1 C:\xampp\htdocs\nxgestao\system\CodeIgniter.php(789): CodeIgniter\Router\Router->handle('NFe/NFeAvulsaSa...')
#2 C:\xampp\htdocs\nxgestao\system\CodeIgniter.php(360): CodeIgniter\CodeIgniter->tryToRouteIt(Object(CodeIgniter\Router\RouteCollection))
#3 C:\xampp\htdocs\nxgestao\system\CodeIgniter.php(308): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\xampp\htdocs\nxgestao\public\index.php(45): CodeIgniter\CodeIgniter->run()
#5 {main}
CRITICAL - 2021-02-01 10:49:39 --> syntax error, unexpected ';', expecting ')'
#0 C:\xampp\htdocs\nxgestao\system\Router\Router.php(192): CodeIgniter\Router\Router->autoRoute('NFe/NFeAvulsaSa...')
#1 C:\xampp\htdocs\nxgestao\system\CodeIgniter.php(789): CodeIgniter\Router\Router->handle('NFe/NFeAvulsaSa...')
#2 C:\xampp\htdocs\nxgestao\system\CodeIgniter.php(360): CodeIgniter\CodeIgniter->tryToRouteIt(Object(CodeIgniter\Router\RouteCollection))
#3 C:\xampp\htdocs\nxgestao\system\CodeIgniter.php(308): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\xampp\htdocs\nxgestao\public\index.php(45): CodeIgniter\CodeIgniter->run()
#5 {main}
CRITICAL - 2021-02-01 10:57:27 --> Undefined property: App\Controllers\NFe::$uf_model
#0 C:\xampp\htdocs\nxgestao\app\Controllers\NFe.php(1465): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Undefined prope...', 'C:\\xampp\\htdocs...', 1465, Array)
#1 C:\xampp\htdocs\nxgestao\system\CodeIgniter.php(914): App\Controllers\NFe->NFeAvulsaSaida()
#2 C:\xampp\htdocs\nxgestao\system\CodeIgniter.php(400): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\NFe))
#3 C:\xampp\htdocs\nxgestao\system\CodeIgniter.php(308): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\xampp\htdocs\nxgestao\public\index.php(45): CodeIgniter\CodeIgniter->run()
#5 {main}
CRITICAL - 2021-02-01 10:58:04 --> Undefined property: App\Controllers\NFe::$municipio_model
#0 C:\xampp\htdocs\nxgestao\app\Controllers\NFe.php(1472): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Undefined prope...', 'C:\\xampp\\htdocs...', 1472, Array)
#1 C:\xampp\htdocs\nxgestao\system\CodeIgniter.php(914): App\Controllers\NFe->NFeAvulsaSaida()
#2 C:\xampp\htdocs\nxgestao\system\CodeIgniter.php(400): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\NFe))
#3 C:\xampp\htdocs\nxgestao\system\CodeIgniter.php(308): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\xampp\htdocs\nxgestao\public\index.php(45): CodeIgniter\CodeIgniter->run()
#5 {main}
CRITICAL - 2021-02-01 10:58:42 --> Undefined index: ie
#0 C:\xampp\htdocs\nxgestao\app\Controllers\NFe.php(1486): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Undefined index...', 'C:\\xampp\\htdocs...', 1486, Array)
#1 C:\xampp\htdocs\nxgestao\system\CodeIgniter.php(914): App\Controllers\NFe->NFeAvulsaSaida()
#2 C:\xampp\htdocs\nxgestao\system\CodeIgniter.php(400): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\NFe))
#3 C:\xampp\htdocs\nxgestao\system\CodeIgniter.php(308): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\xampp\htdocs\nxgestao\public\index.php(45): CodeIgniter\CodeIgniter->run()
#5 {main}
CRITICAL - 2021-02-01 10:59:44 --> Undefined variable: dados_do_destinatario
#0 C:\xampp\htdocs\nxgestao\app\Controllers\NFe.php(785): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Undefined varia...', 'C:\\xampp\\htdocs...', 785, Array)
#1 C:\xampp\htdocs\nxgestao\app\Controllers\NFe.php(1476): App\Controllers\NFe->montaXmlSaidaNFeAvulsa(Array, Array, Array)
#2 C:\xampp\htdocs\nxgestao\system\CodeIgniter.php(914): App\Controllers\NFe->NFeAvulsaSaida()
#3 C:\xampp\htdocs\nxgestao\system\CodeIgniter.php(400): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\NFe))
#4 C:\xampp\htdocs\nxgestao\system\CodeIgniter.php(308): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#5 C:\xampp\htdocs\nxgestao\public\index.php(45): CodeIgniter\CodeIgniter->run()
#6 {main}
CRITICAL - 2021-02-01 11:00:46 --> Undefined variable: dados_do_destinatario
#0 C:\xampp\htdocs\nxgestao\app\Controllers\NFe.php(793): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Undefined varia...', 'C:\\xampp\\htdocs...', 793, Array)
#1 C:\xampp\htdocs\nxgestao\app\Controllers\NFe.php(1484): App\Controllers\NFe->montaXmlSaidaNFeAvulsa(Array, Array, Array)
#2 C:\xampp\htdocs\nxgestao\system\CodeIgniter.php(914): App\Controllers\NFe->NFeAvulsaSaida()
#3 C:\xampp\htdocs\nxgestao\system\CodeIgniter.php(400): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\NFe))
#4 C:\xampp\htdocs\nxgestao\system\CodeIgniter.php(308): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#5 C:\xampp\htdocs\nxgestao\public\index.php(45): CodeIgniter\CodeIgniter->run()
#6 {main}
CRITICAL - 2021-02-01 11:02:27 --> Undefined index: numero
#0 C:\xampp\htdocs\nxgestao\app\Controllers\NFe.php(879): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Undefined index...', 'C:\\xampp\\htdocs...', 879, Array)
#1 C:\xampp\htdocs\nxgestao\app\Controllers\NFe.php(1484): App\Controllers\NFe->montaXmlSaidaNFeAvulsa(Array, Array, Array)
#2 C:\xampp\htdocs\nxgestao\system\CodeIgniter.php(914): App\Controllers\NFe->NFeAvulsaSaida()
#3 C:\xampp\htdocs\nxgestao\system\CodeIgniter.php(400): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\NFe))
#4 C:\xampp\htdocs\nxgestao\system\CodeIgniter.php(308): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#5 C:\xampp\htdocs\nxgestao\public\index.php(45): CodeIgniter\CodeIgniter->run()
#6 {main}
CRITICAL - 2021-02-01 11:03:52 --> Undefined index: codigo
#0 C:\xampp\htdocs\nxgestao\app\Controllers\NFe.php(886): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Undefined index...', 'C:\\xampp\\htdocs...', 886, Array)
#1 C:\xampp\htdocs\nxgestao\app\Controllers\NFe.php(1483): App\Controllers\NFe->montaXmlSaidaNFeAvulsa(Array, Array, Array)
#2 C:\xampp\htdocs\nxgestao\system\CodeIgniter.php(914): App\Controllers\NFe->NFeAvulsaSaida()
#3 C:\xampp\htdocs\nxgestao\system\CodeIgniter.php(400): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\NFe))
#4 C:\xampp\htdocs\nxgestao\system\CodeIgniter.php(308): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#5 C:\xampp\htdocs\nxgestao\public\index.php(45): CodeIgniter\CodeIgniter->run()
#6 {main}
CRITICAL - 2021-02-01 11:04:12 --> Undefined index: municipio
#0 C:\xampp\htdocs\nxgestao\app\Controllers\NFe.php(887): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Undefined index...', 'C:\\xampp\\htdocs...', 887, Array)
#1 C:\xampp\htdocs\nxgestao\app\Controllers\NFe.php(1483): App\Controllers\NFe->montaXmlSaidaNFeAvulsa(Array, Array, Array)
#2 C:\xampp\htdocs\nxgestao\system\CodeIgniter.php(914): App\Controllers\NFe->NFeAvulsaSaida()
#3 C:\xampp\htdocs\nxgestao\system\CodeIgniter.php(400): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\NFe))
#4 C:\xampp\htdocs\nxgestao\system\CodeIgniter.php(308): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#5 C:\xampp\htdocs\nxgestao\public\index.php(45): CodeIgniter\CodeIgniter->run()
#6 {main}
CRITICAL - 2021-02-01 11:04:25 --> Undefined index: uf
#0 C:\xampp\htdocs\nxgestao\app\Controllers\NFe.php(888): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Undefined index...', 'C:\\xampp\\htdocs...', 888, Array)
#1 C:\xampp\htdocs\nxgestao\app\Controllers\NFe.php(1483): App\Controllers\NFe->montaXmlSaidaNFeAvulsa(Array, Array, Array)
#2 C:\xampp\htdocs\nxgestao\system\CodeIgniter.php(914): App\Controllers\NFe->NFeAvulsaSaida()
#3 C:\xampp\htdocs\nxgestao\system\CodeIgniter.php(400): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\NFe))
#4 C:\xampp\htdocs\nxgestao\system\CodeIgniter.php(308): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#5 C:\xampp\htdocs\nxgestao\public\index.php(45): CodeIgniter\CodeIgniter->run()
#6 {main}
CRITICAL - 2021-02-01 11:04:59 --> Undefined variable: dados_do_destinatario
#0 C:\xampp\htdocs\nxgestao\app\Controllers\NFe.php(916): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Undefined varia...', 'C:\\xampp\\htdocs...', 916, Array)
#1 C:\xampp\htdocs\nxgestao\app\Controllers\NFe.php(1483): App\Controllers\NFe->montaXmlSaidaNFeAvulsa(Array, Array, Array)
#2 C:\xampp\htdocs\nxgestao\system\CodeIgniter.php(914): App\Controllers\NFe->NFeAvulsaSaida()
#3 C:\xampp\htdocs\nxgestao\system\CodeIgniter.php(400): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\NFe))
#4 C:\xampp\htdocs\nxgestao\system\CodeIgniter.php(308): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#5 C:\xampp\htdocs\nxgestao\public\index.php(45): CodeIgniter\CodeIgniter->run()
#6 {main}
CRITICAL - 2021-02-01 11:05:33 --> Undefined variable: dados_do_destinatario
#0 C:\xampp\htdocs\nxgestao\app\Controllers\NFe.php(916): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Undefined varia...', 'C:\\xampp\\htdocs...', 916, Array)
#1 C:\xampp\htdocs\nxgestao\app\Controllers\NFe.php(1483): App\Controllers\NFe->montaXmlSaidaNFeAvulsa(Array, Array, Array)
#2 C:\xampp\htdocs\nxgestao\system\CodeIgniter.php(914): App\Controllers\NFe->NFeAvulsaSaida()
#3 C:\xampp\htdocs\nxgestao\system\CodeIgniter.php(400): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\NFe))
#4 C:\xampp\htdocs\nxgestao\system\CodeIgniter.php(308): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#5 C:\xampp\htdocs\nxgestao\public\index.php(45): CodeIgniter\CodeIgniter->run()
#6 {main}
CRITICAL - 2021-02-01 11:05:37 --> Undefined variable: dados_do_destinatario
#0 C:\xampp\htdocs\nxgestao\app\Controllers\NFe.php(916): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Undefined varia...', 'C:\\xampp\\htdocs...', 916, Array)
#1 C:\xampp\htdocs\nxgestao\app\Controllers\NFe.php(1483): App\Controllers\NFe->montaXmlSaidaNFeAvulsa(Array, Array, Array)
#2 C:\xampp\htdocs\nxgestao\system\CodeIgniter.php(914): App\Controllers\NFe->NFeAvulsaSaida()
#3 C:\xampp\htdocs\nxgestao\system\CodeIgniter.php(400): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\NFe))
#4 C:\xampp\htdocs\nxgestao\system\CodeIgniter.php(308): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#5 C:\xampp\htdocs\nxgestao\public\index.php(45): CodeIgniter\CodeIgniter->run()
#6 {main}
CRITICAL - 2021-02-01 11:06:14 --> Undefined variable: dados_do_destinatario
#0 C:\xampp\htdocs\nxgestao\app\Controllers\NFe.php(916): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Undefined varia...', 'C:\\xampp\\htdocs...', 916, Array)
#1 C:\xampp\htdocs\nxgestao\app\Controllers\NFe.php(1483): App\Controllers\NFe->montaXmlSaidaNFeAvulsa(Array, Array, Array)
#2 C:\xampp\htdocs\nxgestao\system\CodeIgniter.php(914): App\Controllers\NFe->NFeAvulsaSaida()
#3 C:\xampp\htdocs\nxgestao\system\CodeIgniter.php(400): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\NFe))
#4 C:\xampp\htdocs\nxgestao\system\CodeIgniter.php(308): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#5 C:\xampp\htdocs\nxgestao\public\index.php(45): CodeIgniter\CodeIgniter->run()
#6 {main}
CRITICAL - 2021-02-01 11:07:50 --> Undefined index: id_transportadora
#0 C:\xampp\htdocs\nxgestao\app\Controllers\NFe.php(1032): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Undefined index...', 'C:\\xampp\\htdocs...', 1032, Array)
#1 C:\xampp\htdocs\nxgestao\app\Controllers\NFe.php(1483): App\Controllers\NFe->montaXmlSaidaNFeAvulsa(Array, Array, Array)
#2 C:\xampp\htdocs\nxgestao\system\CodeIgniter.php(914): App\Controllers\NFe->NFeAvulsaSaida()
#3 C:\xampp\htdocs\nxgestao\system\CodeIgniter.php(400): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\NFe))
#4 C:\xampp\htdocs\nxgestao\system\CodeIgniter.php(308): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#5 C:\xampp\htdocs\nxgestao\public\index.php(45): CodeIgniter\CodeIgniter->run()
#6 {main}
CRITICAL - 2021-02-01 12:05:26 --> Trying to access array offset on value of type null
#0 C:\xampp\htdocs\nxgestao\app\Controllers\NFe.php(1050): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Trying to acces...', 'C:\\xampp\\htdocs...', 1050, Array)
#1 C:\xampp\htdocs\nxgestao\app\Controllers\NFe.php(1483): App\Controllers\NFe->montaXmlSaidaNFeAvulsa(Array, Array, Array)
#2 C:\xampp\htdocs\nxgestao\system\CodeIgniter.php(914): App\Controllers\NFe->NFeAvulsaSaida()
#3 C:\xampp\htdocs\nxgestao\system\CodeIgniter.php(400): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\NFe))
#4 C:\xampp\htdocs\nxgestao\system\CodeIgniter.php(308): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#5 C:\xampp\htdocs\nxgestao\public\index.php(45): CodeIgniter\CodeIgniter->run()
#6 {main}
CRITICAL - 2021-02-01 12:06:19 --> Undefined variable: dados_da_venda
#0 C:\xampp\htdocs\nxgestao\app\Controllers\NFe.php(1084): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Undefined varia...', 'C:\\xampp\\htdocs...', 1084, Array)
#1 C:\xampp\htdocs\nxgestao\app\Controllers\NFe.php(1483): App\Controllers\NFe->montaXmlSaidaNFeAvulsa(Array, Array, Array)
#2 C:\xampp\htdocs\nxgestao\system\CodeIgniter.php(914): App\Controllers\NFe->NFeAvulsaSaida()
#3 C:\xampp\htdocs\nxgestao\system\CodeIgniter.php(400): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\NFe))
#4 C:\xampp\htdocs\nxgestao\system\CodeIgniter.php(308): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#5 C:\xampp\htdocs\nxgestao\public\index.php(45): CodeIgniter\CodeIgniter->run()
#6 {main}
CRITICAL - 2021-02-01 12:12:32 --> Undefined variable: dados_da_venda
#0 C:\xampp\htdocs\nxgestao\app\Controllers\NFe.php(1084): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Undefined varia...', 'C:\\xampp\\htdocs...', 1084, Array)
#1 C:\xampp\htdocs\nxgestao\app\Controllers\NFe.php(1483): App\Controllers\NFe->montaXmlSaidaNFeAvulsa(Array, Array, Array)
#2 C:\xampp\htdocs\nxgestao\system\CodeIgniter.php(914): App\Controllers\NFe->NFeAvulsaSaida()
#3 C:\xampp\htdocs\nxgestao\system\CodeIgniter.php(400): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\NFe))
#4 C:\xampp\htdocs\nxgestao\system\CodeIgniter.php(308): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#5 C:\xampp\htdocs\nxgestao\public\index.php(45): CodeIgniter\CodeIgniter->run()
#6 {main}
CRITICAL - 2021-02-01 12:12:46 --> A non well formed numeric value encountered
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(8, 'A non well form...', 'C:\\xampp\\htdocs...', 5, Array)
#1 C:\xampp\htdocs\nxgestao\app\Helpers\app_helper.php(5): number_format('10,00', 2, '.', '')
#2 C:\xampp\htdocs\nxgestao\app\Controllers\NFe.php(1084): format('10,00')
#3 C:\xampp\htdocs\nxgestao\app\Controllers\NFe.php(1483): App\Controllers\NFe->montaXmlSaidaNFeAvulsa(Array, Array, Array)
#4 C:\xampp\htdocs\nxgestao\system\CodeIgniter.php(914): App\Controllers\NFe->NFeAvulsaSaida()
#5 C:\xampp\htdocs\nxgestao\system\CodeIgniter.php(400): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\NFe))
#6 C:\xampp\htdocs\nxgestao\system\CodeIgniter.php(308): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 C:\xampp\htdocs\nxgestao\public\index.php(45): CodeIgniter\CodeIgniter->run()
#8 {main}
CRITICAL - 2021-02-01 12:13:20 --> Undefined variable: valor_da_nota
#0 C:\xampp\htdocs\nxgestao\app\Controllers\NFe.php(1090): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Undefined varia...', 'C:\\xampp\\htdocs...', 1090, Array)
#1 C:\xampp\htdocs\nxgestao\app\Controllers\NFe.php(1486): App\Controllers\NFe->montaXmlSaidaNFeAvulsa(Array, Array, Array)
#2 C:\xampp\htdocs\nxgestao\system\CodeIgniter.php(914): App\Controllers\NFe->NFeAvulsaSaida()
#3 C:\xampp\htdocs\nxgestao\system\CodeIgniter.php(400): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\NFe))
#4 C:\xampp\htdocs\nxgestao\system\CodeIgniter.php(308): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#5 C:\xampp\htdocs\nxgestao\public\index.php(45): CodeIgniter\CodeIgniter->run()
#6 {main}
CRITICAL - 2021-02-01 12:15:50 --> Undefined index: informacoes_para_fisco
#0 C:\xampp\htdocs\nxgestao\app\Controllers\NFe.php(1109): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Undefined index...', 'C:\\xampp\\htdocs...', 1109, Array)
#1 C:\xampp\htdocs\nxgestao\app\Controllers\NFe.php(1490): App\Controllers\NFe->montaXmlSaidaNFeAvulsa(Array, Array, Array)
#2 C:\xampp\htdocs\nxgestao\system\CodeIgniter.php(914): App\Controllers\NFe->NFeAvulsaSaida()
#3 C:\xampp\htdocs\nxgestao\system\CodeIgniter.php(400): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\NFe))
#4 C:\xampp\htdocs\nxgestao\system\CodeIgniter.php(308): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#5 C:\xampp\htdocs\nxgestao\public\index.php(45): CodeIgniter\CodeIgniter->run()
#6 {main}
CRITICAL - 2021-02-01 12:16:05 --> Undefined index: informacoes_para_fisco
#0 C:\xampp\htdocs\nxgestao\app\Controllers\NFe.php(1109): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Undefined index...', 'C:\\xampp\\htdocs...', 1109, Array)
#1 C:\xampp\htdocs\nxgestao\app\Controllers\NFe.php(1490): App\Controllers\NFe->montaXmlSaidaNFeAvulsa(Array, Array, Array)
#2 C:\xampp\htdocs\nxgestao\system\CodeIgniter.php(914): App\Controllers\NFe->NFeAvulsaSaida()
#3 C:\xampp\htdocs\nxgestao\system\CodeIgniter.php(400): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\NFe))
#4 C:\xampp\htdocs\nxgestao\system\CodeIgniter.php(308): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#5 C:\xampp\htdocs\nxgestao\public\index.php(45): CodeIgniter\CodeIgniter->run()
#6 {main}
CRITICAL - 2021-02-01 12:16:52 --> Undefined index: informacoes_para_fisco
#0 C:\xampp\htdocs\nxgestao\app\Controllers\NFe.php(1109): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Undefined index...', 'C:\\xampp\\htdocs...', 1109, Array)
#1 C:\xampp\htdocs\nxgestao\app\Controllers\NFe.php(1490): App\Controllers\NFe->montaXmlSaidaNFeAvulsa(Array, Array, Array)
#2 C:\xampp\htdocs\nxgestao\system\CodeIgniter.php(914): App\Controllers\NFe->NFeAvulsaSaida()
#3 C:\xampp\htdocs\nxgestao\system\CodeIgniter.php(400): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\NFe))
#4 C:\xampp\htdocs\nxgestao\system\CodeIgniter.php(308): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#5 C:\xampp\htdocs\nxgestao\public\index.php(45): CodeIgniter\CodeIgniter->run()
#6 {main}
CRITICAL - 2021-02-01 12:17:32 --> Undefined index: informacoes_complementares
#0 C:\xampp\htdocs\nxgestao\app\Controllers\NFe.php(1109): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Undefined index...', 'C:\\xampp\\htdocs...', 1109, Array)
#1 C:\xampp\htdocs\nxgestao\app\Controllers\NFe.php(1490): App\Controllers\NFe->montaXmlSaidaNFeAvulsa(Array, Array, Array)
#2 C:\xampp\htdocs\nxgestao\system\CodeIgniter.php(914): App\Controllers\NFe->NFeAvulsaSaida()
#3 C:\xampp\htdocs\nxgestao\system\CodeIgniter.php(400): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\NFe))
#4 C:\xampp\htdocs\nxgestao\system\CodeIgniter.php(308): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#5 C:\xampp\htdocs\nxgestao\public\index.php(45): CodeIgniter\CodeIgniter->run()
#6 {main}
CRITICAL - 2021-02-01 12:24:34 --> Undefined variable: valor_da_nota
#0 C:\xampp\htdocs\nxgestao\app\Controllers\NFe.php(1539): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Undefined varia...', 'C:\\xampp\\htdocs...', 1539, Array)
#1 C:\xampp\htdocs\nxgestao\system\CodeIgniter.php(914): App\Controllers\NFe->NFeAvulsaSaida()
#2 C:\xampp\htdocs\nxgestao\system\CodeIgniter.php(400): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\NFe))
#3 C:\xampp\htdocs\nxgestao\system\CodeIgniter.php(308): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\xampp\htdocs\nxgestao\public\index.php(45): CodeIgniter\CodeIgniter->run()
#5 {main}
CRITICAL - 2021-02-01 12:25:38 --> Cannot add or update a child row: a foreign key constraint fails (`nxgestao`.`nfes`, CONSTRAINT `nfes_id_venda_foreign` FOREIGN KEY (`id_venda`) REFERENCES `vendas` (`id_venda`) ON DELETE CASCADE ON UPDATE CASCADE)
#0 C:\xampp\htdocs\nxgestao\system\Database\MySQLi\Connection.php(331): mysqli->query('INSERT INTO `nf...')
#1 C:\xampp\htdocs\nxgestao\system\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `nf...')
#2 C:\xampp\htdocs\nxgestao\system\Database\BaseConnection.php(666): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `nf...')
#3 C:\xampp\htdocs\nxgestao\system\Database\BaseBuilder.php(2234): CodeIgniter\Database\BaseConnection->query('INSERT INTO `nf...', Array, false)
#4 C:\xampp\htdocs\nxgestao\system\Model.php(729): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\nxgestao\app\Controllers\NFe.php(1551): CodeIgniter\Model->insert(Array)
#6 C:\xampp\htdocs\nxgestao\system\CodeIgniter.php(914): App\Controllers\NFe->NFeAvulsaSaida()
#7 C:\xampp\htdocs\nxgestao\system\CodeIgniter.php(400): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\NFe))
#8 C:\xampp\htdocs\nxgestao\system\CodeIgniter.php(308): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\nxgestao\public\index.php(45): CodeIgniter\CodeIgniter->run()
#10 {main}
CRITICAL - 2021-02-01 12:34:50 --> Undefined property: App\Controllers\NFeAvulsa::$uf_model
#0 C:\xampp\htdocs\nxgestao\app\Controllers\NFeAvulsa.php(112): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Undefined prope...', 'C:\\xampp\\htdocs...', 112, Array)
#1 C:\xampp\htdocs\nxgestao\system\CodeIgniter.php(914): App\Controllers\NFeAvulsa->saida()
#2 C:\xampp\htdocs\nxgestao\system\CodeIgniter.php(400): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\NFeAvulsa))
#3 C:\xampp\htdocs\nxgestao\system\CodeIgniter.php(308): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\xampp\htdocs\nxgestao\public\index.php(45): CodeIgniter\CodeIgniter->run()
#5 {main}
CRITICAL - 2021-02-01 12:37:10 --> Undefined index: nome
#0 C:\xampp\htdocs\nxgestao\app\Views\nfe_avulsa\saida\index.php(325): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Undefined index...', 'C:\\xampp\\htdocs...', 325, Array)
#1 C:\xampp\htdocs\nxgestao\system\View\View.php(246): include('C:\\xampp\\htdocs...')
#2 C:\xampp\htdocs\nxgestao\system\Common.php(1087): CodeIgniter\View\View->render('nfe_avulsa/said...', Array, true)
#3 C:\xampp\htdocs\nxgestao\app\Controllers\NFeAvulsa.php(119): view('nfe_avulsa/said...')
#4 C:\xampp\htdocs\nxgestao\system\CodeIgniter.php(914): App\Controllers\NFeAvulsa->saida()
#5 C:\xampp\htdocs\nxgestao\system\CodeIgniter.php(400): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\NFeAvulsa))
#6 C:\xampp\htdocs\nxgestao\system\CodeIgniter.php(308): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 C:\xampp\htdocs\nxgestao\public\index.php(45): CodeIgniter\CodeIgniter->run()
#8 {main}
CRITICAL - 2021-02-01 12:41:46 --> Undefined index: nome
#0 C:\xampp\htdocs\nxgestao\app\Views\nfe_avulsa\saida\index.php(462): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Undefined index...', 'C:\\xampp\\htdocs...', 462, Array)
#1 C:\xampp\htdocs\nxgestao\system\View\View.php(246): include('C:\\xampp\\htdocs...')
#2 C:\xampp\htdocs\nxgestao\system\Common.php(1087): CodeIgniter\View\View->render('nfe_avulsa/said...', Array, true)
#3 C:\xampp\htdocs\nxgestao\app\Controllers\NFeAvulsa.php(125): view('nfe_avulsa/said...')
#4 C:\xampp\htdocs\nxgestao\system\CodeIgniter.php(914): App\Controllers\NFeAvulsa->saida()
#5 C:\xampp\htdocs\nxgestao\system\CodeIgniter.php(400): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\NFeAvulsa))
#6 C:\xampp\htdocs\nxgestao\system\CodeIgniter.php(308): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 C:\xampp\htdocs\nxgestao\public\index.php(45): CodeIgniter\CodeIgniter->run()
#8 {main}
CRITICAL - 2021-02-01 12:48:27 --> Undefined index: transportadora
#0 C:\xampp\htdocs\nxgestao\app\Views\nfe_avulsa\saida\index.php(447): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Undefined index...', 'C:\\xampp\\htdocs...', 447, Array)
#1 C:\xampp\htdocs\nxgestao\system\View\View.php(246): include('C:\\xampp\\htdocs...')
#2 C:\xampp\htdocs\nxgestao\system\Common.php(1087): CodeIgniter\View\View->render('nfe_avulsa/said...', Array, true)
#3 C:\xampp\htdocs\nxgestao\app\Controllers\NFeAvulsa.php(132): view('nfe_avulsa/said...')
#4 C:\xampp\htdocs\nxgestao\system\CodeIgniter.php(914): App\Controllers\NFeAvulsa->saida()
#5 C:\xampp\htdocs\nxgestao\system\CodeIgniter.php(400): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\NFeAvulsa))
#6 C:\xampp\htdocs\nxgestao\system\CodeIgniter.php(308): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 C:\xampp\htdocs\nxgestao\public\index.php(45): CodeIgniter\CodeIgniter->run()
#8 {main}
CRITICAL - 2021-02-01 12:49:36 --> Undefined index: qVol
#0 C:\xampp\htdocs\nxgestao\app\Controllers\NFe.php(1077): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Undefined index...', 'C:\\xampp\\htdocs...', 1077, Array)
#1 C:\xampp\htdocs\nxgestao\app\Controllers\NFe.php(1493): App\Controllers\NFe->montaXmlSaidaNFeAvulsa(Array, Array, Array)
#2 C:\xampp\htdocs\nxgestao\system\CodeIgniter.php(914): App\Controllers\NFe->NFeAvulsaSaida()
#3 C:\xampp\htdocs\nxgestao\system\CodeIgniter.php(400): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\NFe))
#4 C:\xampp\htdocs\nxgestao\system\CodeIgniter.php(308): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#5 C:\xampp\htdocs\nxgestao\public\index.php(45): CodeIgniter\CodeIgniter->run()
#6 {main}
CRITICAL - 2021-02-01 12:51:19 --> Trying to access array offset on value of type null
#0 C:\xampp\htdocs\nxgestao\app\Controllers\ControleFiscal.php(152): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Trying to acces...', 'C:\\xampp\\htdocs...', 152, Array)
#1 C:\xampp\htdocs\nxgestao\system\CodeIgniter.php(914): App\Controllers\ControleFiscal->baixaXML('7', '1')
#2 C:\xampp\htdocs\nxgestao\system\CodeIgniter.php(400): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\ControleFiscal))
#3 C:\xampp\htdocs\nxgestao\system\CodeIgniter.php(308): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\xampp\htdocs\nxgestao\public\index.php(45): CodeIgniter\CodeIgniter->run()
#5 {main}
CRITICAL - 2021-02-01 15:39:50 --> Invalid file: nfe_avulsa/devolucao/index.php
#0 C:\xampp\htdocs\nxgestao\system\View\View.php(228): CodeIgniter\Exceptions\FrameworkException::forInvalidFile('nfe_avulsa/devo...')
#1 C:\xampp\htdocs\nxgestao\system\Common.php(1087): CodeIgniter\View\View->render('nfe_avulsa/devo...', Array, true)
#2 C:\xampp\htdocs\nxgestao\app\Controllers\NFeAvulsa.php(179): view('nfe_avulsa/devo...')
#3 C:\xampp\htdocs\nxgestao\system\CodeIgniter.php(914): App\Controllers\NFeAvulsa->devolucao()
#4 C:\xampp\htdocs\nxgestao\system\CodeIgniter.php(400): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\NFeAvulsa))
#5 C:\xampp\htdocs\nxgestao\system\CodeIgniter.php(308): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#6 C:\xampp\htdocs\nxgestao\public\index.php(45): CodeIgniter\CodeIgniter->run()
#7 {main}
CRITICAL - 2021-02-01 16:05:39 --> Uncaught ErrorException: Array to string conversion in C:\xampp\htdocs\nxgestao\app\Controllers\NFe.php:1545
Stack trace:
#0 C:\xampp\htdocs\nxgestao\app\Controllers\NFe.php(1545): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Array to string...', 'C:\\xampp\\htdocs...', 1545, Array)
#1 C:\xampp\htdocs\nxgestao\app\Controllers\NFe.php(2002): App\Controllers\NFe->montaXmlDevolucaoNFeAvulsa(Array, Array, Array)
#2 C:\xampp\htdocs\nxgestao\system\CodeIgniter.php(914): App\Controllers\NFe->NFeAvulsaDevolucao()
#3 C:\xampp\htdocs\nxgestao\system\CodeIgniter.php(400): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\NFe))
#4 C:\xampp\htdocs\nxgestao\system\CodeIgniter.php(308): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#5 C:\xampp\htdocs\nxgestao\public\index.php(45): CodeIgniter\CodeIgniter->run()
#6 {main}
  thrown
#0 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
#1 {main}
CRITICAL - 2021-02-01 16:06:28 --> Uncaught ErrorException: Array to string conversion in C:\xampp\htdocs\nxgestao\app\Controllers\NFe.php:1545
Stack trace:
#0 C:\xampp\htdocs\nxgestao\app\Controllers\NFe.php(1545): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Array to string...', 'C:\\xampp\\htdocs...', 1545, Array)
#1 C:\xampp\htdocs\nxgestao\app\Controllers\NFe.php(2002): App\Controllers\NFe->montaXmlDevolucaoNFeAvulsa(Array, Array, Array)
#2 C:\xampp\htdocs\nxgestao\system\CodeIgniter.php(914): App\Controllers\NFe->NFeAvulsaDevolucao()
#3 C:\xampp\htdocs\nxgestao\system\CodeIgniter.php(400): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\NFe))
#4 C:\xampp\htdocs\nxgestao\system\CodeIgniter.php(308): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#5 C:\xampp\htdocs\nxgestao\public\index.php(45): CodeIgniter\CodeIgniter->run()
#6 {main}
  thrown
#0 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
#1 {main}
CRITICAL - 2021-02-01 16:06:52 --> Call to a member function getElementsByTagName() on null
#0 C:\xampp\htdocs\nxgestao\app\ThirdParty\sped-nfe\vendor\nfephp-org\sped-nfe\src\Make.php(441): NFePHP\NFe\Make->checkNFeKey(Object(NFePHP\Common\DOMImproved))
#1 C:\xampp\htdocs\nxgestao\app\ThirdParty\sped-nfe\vendor\nfephp-org\sped-nfe\src\Make.php(370): NFePHP\NFe\Make->monta()
#2 C:\xampp\htdocs\nxgestao\app\ThirdParty\sped-nfe\vendor\nfephp-org\sped-nfe\src\Make.php(341): NFePHP\NFe\Make->montaNFe()
#3 C:\xampp\htdocs\nxgestao\app\Controllers\NFe.php(1170): NFePHP\NFe\Make->getXML()
#4 C:\xampp\htdocs\nxgestao\app\Controllers\NFe.php(2004): App\Controllers\NFe->montaXmlDevolucaoNFeAvulsa(Array, Array, Array)
#5 C:\xampp\htdocs\nxgestao\system\CodeIgniter.php(914): App\Controllers\NFe->NFeAvulsaDevolucao()
#6 C:\xampp\htdocs\nxgestao\system\CodeIgniter.php(400): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\NFe))
#7 C:\xampp\htdocs\nxgestao\system\CodeIgniter.php(308): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\nxgestao\public\index.php(45): CodeIgniter\CodeIgniter->run()
#9 {main}
CRITICAL - 2021-02-01 16:07:29 --> Uncaught ErrorException: Array to string conversion in C:\xampp\htdocs\nxgestao\app\Controllers\NFe.php:1545
Stack trace:
#0 C:\xampp\htdocs\nxgestao\app\Controllers\NFe.php(1545): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Array to string...', 'C:\\xampp\\htdocs...', 1545, Array)
#1 C:\xampp\htdocs\nxgestao\app\Controllers\NFe.php(2002): App\Controllers\NFe->montaXmlDevolucaoNFeAvulsa(Array, Array, Array)
#2 C:\xampp\htdocs\nxgestao\system\CodeIgniter.php(914): App\Controllers\NFe->NFeAvulsaDevolucao()
#3 C:\xampp\htdocs\nxgestao\system\CodeIgniter.php(400): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\NFe))
#4 C:\xampp\htdocs\nxgestao\system\CodeIgniter.php(308): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#5 C:\xampp\htdocs\nxgestao\public\index.php(45): CodeIgniter\CodeIgniter->run()
#6 {main}
  thrown
#0 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
#1 {main}
CRITICAL - 2021-02-01 16:09:32 --> Uncaught ErrorException: Array to string conversion in C:\xampp\htdocs\nxgestao\app\Controllers\NFe.php:1545
Stack trace:
#0 C:\xampp\htdocs\nxgestao\app\Controllers\NFe.php(1545): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Array to string...', 'C:\\xampp\\htdocs...', 1545, Array)
#1 C:\xampp\htdocs\nxgestao\app\Controllers\NFe.php(2002): App\Controllers\NFe->montaXmlDevolucaoNFeAvulsa(Array, Array, Array)
#2 C:\xampp\htdocs\nxgestao\system\CodeIgniter.php(914): App\Controllers\NFe->NFeAvulsaDevolucao()
#3 C:\xampp\htdocs\nxgestao\system\CodeIgniter.php(400): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\NFe))
#4 C:\xampp\htdocs\nxgestao\system\CodeIgniter.php(308): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#5 C:\xampp\htdocs\nxgestao\public\index.php(45): CodeIgniter\CodeIgniter->run()
#6 {main}
  thrown
#0 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
#1 {main}
CRITICAL - 2021-02-01 16:11:32 --> Call to undefined function App\Controllers\removeMascaras()
#0 C:\xampp\htdocs\nxgestao\app\Controllers\NFe.php(2002): App\Controllers\NFe->montaXmlDevolucaoNFeAvulsa(Array, Array, Array)
#1 C:\xampp\htdocs\nxgestao\system\CodeIgniter.php(914): App\Controllers\NFe->NFeAvulsaDevolucao()
#2 C:\xampp\htdocs\nxgestao\system\CodeIgniter.php(400): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\NFe))
#3 C:\xampp\htdocs\nxgestao\system\CodeIgniter.php(308): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\xampp\htdocs\nxgestao\public\index.php(45): CodeIgniter\CodeIgniter->run()
#5 {main}
CRITICAL - 2021-02-01 16:15:49 --> Uncaught ErrorException: Array to string conversion in C:\xampp\htdocs\nxgestao\app\Controllers\NFe.php:1545
Stack trace:
#0 C:\xampp\htdocs\nxgestao\app\Controllers\NFe.php(1545): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Array to string...', 'C:\\xampp\\htdocs...', 1545, Array)
#1 C:\xampp\htdocs\nxgestao\app\Controllers\NFe.php(2002): App\Controllers\NFe->montaXmlDevolucaoNFeAvulsa(Array, Array, Array)
#2 C:\xampp\htdocs\nxgestao\system\CodeIgniter.php(914): App\Controllers\NFe->NFeAvulsaDevolucao()
#3 C:\xampp\htdocs\nxgestao\system\CodeIgniter.php(400): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\NFe))
#4 C:\xampp\htdocs\nxgestao\system\CodeIgniter.php(308): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#5 C:\xampp\htdocs\nxgestao\public\index.php(45): CodeIgniter\CodeIgniter->run()
#6 {main}
  thrown
#0 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
#1 {main}
CRITICAL - 2021-02-01 16:20:30 --> Uncaught ErrorException: Array to string conversion in C:\xampp\htdocs\nxgestao\app\Controllers\NFe.php:1544
Stack trace:
#0 C:\xampp\htdocs\nxgestao\app\Controllers\NFe.php(1544): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Array to string...', 'C:\\xampp\\htdocs...', 1544, Array)
#1 C:\xampp\htdocs\nxgestao\app\Controllers\NFe.php(2001): App\Controllers\NFe->montaXmlDevolucaoNFeAvulsa(Array, Array, Array)
#2 C:\xampp\htdocs\nxgestao\system\CodeIgniter.php(914): App\Controllers\NFe->NFeAvulsaDevolucao()
#3 C:\xampp\htdocs\nxgestao\system\CodeIgniter.php(400): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\NFe))
#4 C:\xampp\htdocs\nxgestao\system\CodeIgniter.php(308): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#5 C:\xampp\htdocs\nxgestao\public\index.php(45): CodeIgniter\CodeIgniter->run()
#6 {main}
  thrown
#0 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
#1 {main}
CRITICAL - 2021-02-01 16:20:47 --> Uncaught ErrorException: Array to string conversion in C:\xampp\htdocs\nxgestao\app\Controllers\NFe.php:1544
Stack trace:
#0 C:\xampp\htdocs\nxgestao\app\Controllers\NFe.php(1544): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Array to string...', 'C:\\xampp\\htdocs...', 1544, Array)
#1 C:\xampp\htdocs\nxgestao\app\Controllers\NFe.php(2001): App\Controllers\NFe->montaXmlDevolucaoNFeAvulsa(Array, Array, Array)
#2 C:\xampp\htdocs\nxgestao\system\CodeIgniter.php(914): App\Controllers\NFe->NFeAvulsaDevolucao()
#3 C:\xampp\htdocs\nxgestao\system\CodeIgniter.php(400): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\NFe))
#4 C:\xampp\htdocs\nxgestao\system\CodeIgniter.php(308): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#5 C:\xampp\htdocs\nxgestao\public\index.php(45): CodeIgniter\CodeIgniter->run()
#6 {main}
  thrown
#0 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
#1 {main}
CRITICAL - 2021-02-01 16:53:42 --> syntax error, unexpected 'return' (T_RETURN), expecting function (T_FUNCTION) or const (T_CONST)
#0 C:\xampp\htdocs\nxgestao\system\Router\Router.php(192): CodeIgniter\Router\Router->autoRoute('boletos')
#1 C:\xampp\htdocs\nxgestao\system\CodeIgniter.php(789): CodeIgniter\Router\Router->handle('boletos')
#2 C:\xampp\htdocs\nxgestao\system\CodeIgniter.php(360): CodeIgniter\CodeIgniter->tryToRouteIt(Object(CodeIgniter\Router\RouteCollection))
#3 C:\xampp\htdocs\nxgestao\system\CodeIgniter.php(308): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\xampp\htdocs\nxgestao\public\index.php(45): CodeIgniter\CodeIgniter->run()
#5 {main}
CRITICAL - 2021-02-01 17:58:13 --> Invalid file: admin/boletos/create.php
#0 C:\xampp\htdocs\nxgestao\system\View\View.php(228): CodeIgniter\Exceptions\FrameworkException::forInvalidFile('admin/boletos/c...')
#1 C:\xampp\htdocs\nxgestao\system\Common.php(1087): CodeIgniter\View\View->render('admin/boletos/c...', Array, true)
#2 C:\xampp\htdocs\nxgestao\app\Controllers\Boletos.php(77): view('admin/boletos/c...')
#3 C:\xampp\htdocs\nxgestao\system\CodeIgniter.php(914): App\Controllers\Boletos->create()
#4 C:\xampp\htdocs\nxgestao\system\CodeIgniter.php(400): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Boletos))
#5 C:\xampp\htdocs\nxgestao\system\CodeIgniter.php(308): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#6 C:\xampp\htdocs\nxgestao\public\index.php(45): CodeIgniter\CodeIgniter->run()
#7 {main}
CRITICAL - 2021-02-01 18:19:16 --> Undefined index: valor
#0 C:\xampp\htdocs\nxgestao\app\Controllers\Boletos.php(86): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Undefined index...', 'C:\\xampp\\htdocs...', 86, Array)
#1 C:\xampp\htdocs\nxgestao\system\CodeIgniter.php(914): App\Controllers\Boletos->store()
#2 C:\xampp\htdocs\nxgestao\system\CodeIgniter.php(400): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Boletos))
#3 C:\xampp\htdocs\nxgestao\system\CodeIgniter.php(308): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\xampp\htdocs\nxgestao\public\index.php(45): CodeIgniter\CodeIgniter->run()
#5 {main}
CRITICAL - 2021-02-01 18:29:27 --> Table 'nxgestao.boletos' doesn't exist
#0 C:\xampp\htdocs\nxgestao\system\Database\MySQLi\Connection.php(331): mysqli->query('INSERT INTO `bo...')
#1 C:\xampp\htdocs\nxgestao\system\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `bo...')
#2 C:\xampp\htdocs\nxgestao\system\Database\BaseConnection.php(666): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `bo...')
#3 C:\xampp\htdocs\nxgestao\system\Database\BaseBuilder.php(2234): CodeIgniter\Database\BaseConnection->query('INSERT INTO `bo...', Array, false)
#4 C:\xampp\htdocs\nxgestao\system\Model.php(729): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\nxgestao\app\Controllers\Boletos.php(177): CodeIgniter\Model->insert(Array)
#6 C:\xampp\htdocs\nxgestao\system\CodeIgniter.php(914): App\Controllers\Boletos->store()
#7 C:\xampp\htdocs\nxgestao\system\CodeIgniter.php(400): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Boletos))
#8 C:\xampp\htdocs\nxgestao\system\CodeIgniter.php(308): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\nxgestao\public\index.php(45): CodeIgniter\CodeIgniter->run()
#10 {main}
CRITICAL - 2021-02-01 18:29:48 --> Column 'id_empresa' cannot be null
#0 C:\xampp\htdocs\nxgestao\system\Database\MySQLi\Connection.php(331): mysqli->query('INSERT INTO `bo...')
#1 C:\xampp\htdocs\nxgestao\system\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `bo...')
#2 C:\xampp\htdocs\nxgestao\system\Database\BaseConnection.php(666): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `bo...')
#3 C:\xampp\htdocs\nxgestao\system\Database\BaseBuilder.php(2234): CodeIgniter\Database\BaseConnection->query('INSERT INTO `bo...', Array, false)
#4 C:\xampp\htdocs\nxgestao\system\Model.php(729): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\nxgestao\app\Controllers\Boletos.php(177): CodeIgniter\Model->insert(Array)
#6 C:\xampp\htdocs\nxgestao\system\CodeIgniter.php(914): App\Controllers\Boletos->store()
#7 C:\xampp\htdocs\nxgestao\system\CodeIgniter.php(400): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Boletos))
#8 C:\xampp\htdocs\nxgestao\system\CodeIgniter.php(308): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\nxgestao\public\index.php(45): CodeIgniter\CodeIgniter->run()
#10 {main}
CRITICAL - 2021-02-01 18:34:49 --> Unknown column 'id_empresa' in 'where clause'
#0 C:\xampp\htdocs\nxgestao\system\Database\MySQLi\Connection.php(331): mysqli->query('SELECT *\nFROM `...')
#1 C:\xampp\htdocs\nxgestao\system\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *\nFROM `...')
#2 C:\xampp\htdocs\nxgestao\system\Database\BaseConnection.php(666): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *\nFROM `...')
#3 C:\xampp\htdocs\nxgestao\system\Database\BaseBuilder.php(1845): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\nxgestao\system\Model.php(445): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\nxgestao\app\Controllers\Boletos.php(58): CodeIgniter\Model->findAll()
#6 C:\xampp\htdocs\nxgestao\system\CodeIgniter.php(914): App\Controllers\Boletos->index()
#7 C:\xampp\htdocs\nxgestao\system\CodeIgniter.php(400): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Boletos))
#8 C:\xampp\htdocs\nxgestao\system\CodeIgniter.php(308): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\nxgestao\public\index.php(45): CodeIgniter\CodeIgniter->run()
#10 {main}
CRITICAL - 2021-02-01 19:57:25 --> Undefined property: stdClass::$_embedded
#0 C:\xampp\htdocs\nxgestao\app\Controllers\Boletos.php(169): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Undefined prope...', 'C:\\xampp\\htdocs...', 169, Array)
#1 C:\xampp\htdocs\nxgestao\system\CodeIgniter.php(914): App\Controllers\Boletos->store()
#2 C:\xampp\htdocs\nxgestao\system\CodeIgniter.php(400): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Boletos))
#3 C:\xampp\htdocs\nxgestao\system\CodeIgniter.php(308): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\xampp\htdocs\nxgestao\public\index.php(45): CodeIgniter\CodeIgniter->run()
#5 {main}
CRITICAL - 2021-02-01 19:57:54 --> Undefined property: stdClass::$_embedded
#0 C:\xampp\htdocs\nxgestao\app\Controllers\Boletos.php(169): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Undefined prope...', 'C:\\xampp\\htdocs...', 169, Array)
#1 C:\xampp\htdocs\nxgestao\system\CodeIgniter.php(914): App\Controllers\Boletos->store()
#2 C:\xampp\htdocs\nxgestao\system\CodeIgniter.php(400): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Boletos))
#3 C:\xampp\htdocs\nxgestao\system\CodeIgniter.php(308): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\xampp\htdocs\nxgestao\public\index.php(45): CodeIgniter\CodeIgniter->run()
#5 {main}
CRITICAL - 2021-02-01 19:58:15 --> Undefined property: stdClass::$_embedded
#0 C:\xampp\htdocs\nxgestao\app\Controllers\Boletos.php(169): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Undefined prope...', 'C:\\xampp\\htdocs...', 169, Array)
#1 C:\xampp\htdocs\nxgestao\system\CodeIgniter.php(914): App\Controllers\Boletos->store()
#2 C:\xampp\htdocs\nxgestao\system\CodeIgniter.php(400): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Boletos))
#3 C:\xampp\htdocs\nxgestao\system\CodeIgniter.php(308): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\xampp\htdocs\nxgestao\public\index.php(45): CodeIgniter\CodeIgniter->run()
#5 {main}
CRITICAL - 2021-02-01 19:58:23 --> Undefined property: stdClass::$_embedded
#0 C:\xampp\htdocs\nxgestao\app\Controllers\Boletos.php(169): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Undefined prope...', 'C:\\xampp\\htdocs...', 169, Array)
#1 C:\xampp\htdocs\nxgestao\system\CodeIgniter.php(914): App\Controllers\Boletos->store()
#2 C:\xampp\htdocs\nxgestao\system\CodeIgniter.php(400): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Boletos))
#3 C:\xampp\htdocs\nxgestao\system\CodeIgniter.php(308): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\xampp\htdocs\nxgestao\public\index.php(45): CodeIgniter\CodeIgniter->run()
#5 {main}
CRITICAL - 2021-02-01 20:02:35 --> Trying to get property 'message' of non-object
#0 C:\xampp\htdocs\nxgestao\app\Controllers\Boletos.php(167): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Trying to get p...', 'C:\\xampp\\htdocs...', 167, Array)
#1 C:\xampp\htdocs\nxgestao\system\CodeIgniter.php(914): App\Controllers\Boletos->store()
#2 C:\xampp\htdocs\nxgestao\system\CodeIgniter.php(400): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Boletos))
#3 C:\xampp\htdocs\nxgestao\system\CodeIgniter.php(308): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\xampp\htdocs\nxgestao\public\index.php(45): CodeIgniter\CodeIgniter->run()
#5 {main}
CRITICAL - 2021-02-01 20:05:53 --> Undefined property: stdClass::$_embedded
#0 C:\xampp\htdocs\nxgestao\app\Controllers\Boletos.php(200): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Undefined prope...', 'C:\\xampp\\htdocs...', 200, Array)
#1 C:\xampp\htdocs\nxgestao\system\CodeIgniter.php(914): App\Controllers\Boletos->store()
#2 C:\xampp\htdocs\nxgestao\system\CodeIgniter.php(400): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Boletos))
#3 C:\xampp\htdocs\nxgestao\system\CodeIgniter.php(308): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\xampp\htdocs\nxgestao\public\index.php(45): CodeIgniter\CodeIgniter->run()
#5 {main}
CRITICAL - 2021-02-01 20:06:45 --> Undefined property: stdClass::$_embedded
#0 C:\xampp\htdocs\nxgestao\app\Controllers\Boletos.php(200): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Undefined prope...', 'C:\\xampp\\htdocs...', 200, Array)
#1 C:\xampp\htdocs\nxgestao\system\CodeIgniter.php(914): App\Controllers\Boletos->store()
#2 C:\xampp\htdocs\nxgestao\system\CodeIgniter.php(400): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Boletos))
#3 C:\xampp\htdocs\nxgestao\system\CodeIgniter.php(308): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#4 C:\xampp\htdocs\nxgestao\public\index.php(45): CodeIgniter\CodeIgniter->run()
#5 {main}
CRITICAL - 2021-02-01 20:09:14 --> Trying to access array offset on value of type null
#0 C:\xampp\htdocs\nxgestao\app\Models\ControleDeAcessoModel.php(105): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Trying to acces...', 'C:\\xampp\\htdocs...', 105, Array)
#1 C:\xampp\htdocs\nxgestao\app\Controllers\Caixas.php(47): App\Models\ControleDeAcessoModel->verificaPermissao('caixas')
#2 C:\xampp\htdocs\nxgestao\system\CodeIgniter.php(886): App\Controllers\Caixas->__construct()
#3 C:\xampp\htdocs\nxgestao\system\CodeIgniter.php(395): CodeIgniter\CodeIgniter->createController()
#4 C:\xampp\htdocs\nxgestao\system\CodeIgniter.php(308): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#5 C:\xampp\htdocs\nxgestao\public\index.php(45): CodeIgniter\CodeIgniter->run()
#6 {main}
CRITICAL - 2021-02-01 20:13:03 --> Undefined index: item
#0 C:\xampp\htdocs\nxgestao\app\Views\templates\footer.php(104): CodeIgniter\Debug\Exceptions->errorHandler(8, 'Undefined index...', 'C:\\xampp\\htdocs...', 104, Array)
#1 C:\xampp\htdocs\nxgestao\system\View\View.php(246): include('C:\\xampp\\htdocs...')
#2 C:\xampp\htdocs\nxgestao\system\Common.php(1087): CodeIgniter\View\View->render('templates/foote...', Array, true)
#3 C:\xampp\htdocs\nxgestao\app\Controllers\Boletos.php(59): view('templates/foote...')
#4 C:\xampp\htdocs\nxgestao\system\CodeIgniter.php(914): App\Controllers\Boletos->index()
#5 C:\xampp\htdocs\nxgestao\system\CodeIgniter.php(400): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Boletos))
#6 C:\xampp\htdocs\nxgestao\system\CodeIgniter.php(308): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 C:\xampp\htdocs\nxgestao\public\index.php(45): CodeIgniter\CodeIgniter->run()
#8 {main}
